DevIncubator-Autopark/Program.cs

```diff
@@ -2,35 +2,32 @@
 {
     class Program
     {
+        private static readonly string CsvFilesPath = @$"..\..\..\FileData\";
```

Comment:
Just imagine this project will go into production. This line will not work because production app will have compiled code only. So set for your csv file copy on compile and use base directory only.

Reply:
fixed